packages/react-aria-components/docs/Tree.mdx

```diff
@@ -237,6 +242,517 @@ let items = [
+`Tree` helps achieve accessible and interactive tree components that can be styled as needed.
+
+* **Item selection** – Single or multiple selection, with optional checkboxes, disabled items, and both `toggle` and `replace` selection behaviors.
+* **Interactive children** – Tree items may include interactive elements such as buttons, menus, etc.
+* **Actions** – Items support optional actions such as navigation via click, tap, double click, or <Keyboard>Enter</Keyboard> key.
+* **Keyboard navigation** – Tree items and focusable children can be navigated using the arrow keys, along with page up/down, home/end, etc. Typeahead, auto scrolling, and selection modifier keys are supported as well.
+* **Touch friendly** – Selection and actions adapt their behavior depending on the device. For example, selection is activated via long press on touch when item actions are present.
+* **Accessible** – Follows the [ARIA grid pattern](https://www.w3.org/WAI/ARIA/apg/patterns/treeview/), with additional selection announcements via an ARIA live region. Extensively tested across many devices and [assistive technologies](accessibility.html#testing) to ensure announcements and behaviors are consistent.
+
+## Anatomy
+
+<Anatomy role="img" aria-label="Anatomy diagram of a Tree, consisting of multiple rows. Each row in the table body contains a selection checkbox, an expand toggle button, and the name of the item." />
+
+A Tree consists of a container element, with items containing data inside. The items within a tree may contain focusable elements or plain text content. Each item may also contain a button to toggle the expandable state of that item.
+
+If the tree supports item selection, each item can optionally include a selection checkbox.
+
+
+### Concepts
+
+`Tree` makes use of the following concepts:
+
+<section className={styles.cardGroup} data-size="small">
+
+<ExampleCard
+  url="collections.html"
+  title="Collections"
+  description="Defining collections of items, async loading, and updating items over time.">
+  <Collections />
+</ExampleCard>
+
+<ExampleCard
+  url="selection.html"
+  title="Selection"
+  description="Interactions and data structures to represent selection.">
+  <Selection />
+</ExampleCard>
+
+</section>
+
+### Composed components
+
+A `Tree` uses the following components, which may also be used standalone or reused in other components.
+
+<section className={styles.cardGroup} data-size="small">
+
+<ExampleCard
+  url="Checkbox.html"
+  title="Checkbox"
+  description="A checkbox allows a user to select an individual option.">
+  <Checkbox />
+</ExampleCard>
+
+<ExampleCard
+  url="Button.html"
+  title="Button"
+  description="A button allows a user to perform an action.">
+  <Button />
+</ExampleCard>
+
+</section>
+
+## Examples
+
+<ExampleList tag="tree" />
```

Comment:
Chevron does not pass contrast in light mode. Maybe the background and text color is what should be changed in light mode.

Reply:
can you provide a screenshot? I'm not noticing anything and axe dev tools isn't showing me anything

Reply:
<img width="1070" alt="Screenshot 2025-02-14 at 11 14 53 AM" src="https://github.com/user-attachments/assets/c514e867-6e9d-4a5e-b2e3-ee69d52e6c0b" />

Reply:
Thanks, I believe I've met the contrast requirements now, selection is 3.74:1 and text on selection is now 4.77:1